charts/falco/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.3.1
-appVersion: "0.37.1"
+version: 4.4.0
+appVersion: "0.38-rc3"
```

```suggestion
appVersion: "0.38.0-rc3"
```

Will be replaced with `"0.38.0"` in the end though

---

I'll apply `0.38.0`